mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule GraphQL.Plug.Mixfile do
 
      {:cowboy, "~> 1.0"},
      {:plug, "~> 0.14 or ~> 1.0"},
-     {:poison, "~> 1.5 or ~> 2.0"},
+     {:poison, "~> 1.5 or ~> 2.0 or ~> 3.0"},
```

Comment:
I'm wondering if 1.5 can be removed?

Also Jose updated from 2.0 to 2.2 in order to support some calendaring. https://github.com/phoenixframework/phoenix/commit/655f45bf9cf6cc0b1d85af3d1c9c43afbcacaf1a#diff-6023be6004fce4718dad3dafb576d258